static/cart.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="index.css">
+  <title>Document</title>
+</head>
+<body>
+  <div class="app layout">
+    <div class="layout__header header">
+      <div class="header__cart-info">
+        <button class="cart-info-cta__button"><</button>
+        <h1 class="cart-info__title">Tu Carrito (4)</h1>
+      </div>
```

Comment:
Please checkout the following feedback. If you do not understand something or you do not agree please let me know!

### 1. Top Bar

You should think of these 3 html tags as a UI element. We could give it a name and call it Top Bar. This would be considered a [molecule](https://bradfrost.com/blog/post/atomic-web-design/#molecules) if we were following Atomic Design principles;

> Molecules are groups of atoms bonded together and are the smallest fundamental units of a compound

If you think about it, our **Top Bar's anatomy** is:
* A **navigation icon** placed on the far left
* A centered **title** with the amount of items in our cart

This means that our Top Bar molecule is made up of 2 atoms (navigation icon and a title).

Here is an example: [material design's Tob Bar component](https://material.io/components/app-bars-top#anatomy). 

### 2. CSS class naming

All 3 CSS classes used here do not communicate the ideas described above (point 1), because they do not have naming cohesion and are not following BEM correctly.

If you think of Top Bar as a unit (molecule) and try to use [BEM](http://getbem.com/naming/) for class names, you will realise:

* The div acting as a container is the **Block** (example: `top-bar`)
* Both the navigation icon and the title are **Elements** (examples: `top-bar__navigation-icon` and `top-bar__title`)

This means that our class names should change to something like:
```html
      <div class="top-bar">
        <button class="top-bar__navigation-icon"><</button>
        <h1 class="top-bar__title">Tu Carrito (4)</h1>
      </div>
```

By doing this, just by looking to our code everything looks like a group. It not only seems like every element is part of `top-bar` but also we provide a self-explanatory explicit name.

### 3. Apply Single Responsibility Principle also in CSS classes.

So far we have just talked about UI elements, Atomic Design and BEM. But there are also good coding practices (or software principles) that we can use here in CSS:

#### 3.1 DRY (Don't repeat yourself)
> Don't repeat yourself (DRY, or sometimes do not repeat yourself) is a principle of software development aimed at reducing repetition of software patterns, replacing it with abstractions or using data normalization to avoid redundancy. (wikipedia)

This principle applied to CSS means that we should try to reduce code repetition by reusing CSS classes. This is exactly what [Atomic Design](https://bradfrost.com/blog/post/atomic-web-design/) is all about...

> Interfaces are made up of smaller components. This means we can break entire interfaces down into fundamental building blocks and work up from there. That’s the basic gist of atomic design.

This means that we should try to break our UI into small components (atoms) that we can reuse in multiple places to build bigger blocks (elements and organisms).

As we already said, our `top-bar` is made of a nav icon and a **title**. If you checkout the wireframes you will see that this kind of title is used in other places (desktop design) and it will probably be used in other pages.

![image](https://user-images.githubusercontent.com/17853818/104794764-18f99a00-57aa-11eb-99f1-e96a0de248e0.png)

Actually, on the css file (lines 139 and 144) you will realise that `cart-info__title` and `title-3` actually share the same `font-size`. We could say that top-bar's title is of type `title-3`.

![image](https://user-images.githubusercontent.com/17853818/104794794-53633700-57aa-11eb-8744-8e68406060f2.png)

We can remove font-size from `top-bar__title` and use `title-3` class on our html tag:

```html
      <div class="top-bar">
        <button class="top-bar__navigation-icon"><</button>
        <h1 class="top-bar__title title-3">Tu Carrito (4)</h1>
      </div>
```

```css
.top-bar__title {
  align-self: center;
  justify-self: auto;
  margin: 0 auto;
  font-weight: 600;
}

.title-3 {
  font-size: 18px;
}
```

#### 3.1 Single Responsibility Principle

When we start reusing code, we inevitably divide css attributes like in our previous example; `title-3` applies `font-size` to our element and the rest is applied by `top-bar__title`.

Then a question arises: **What css attributes go in which class?**

This is not an easy question to answer, but we can try to apply [the Single Responsibility Principle](https://stackify.com/solid-design-principles/)... I would suggest the following:

1. `title-3` css class is used to define the title as an atom, independ of the top bar. This means it should have all css attributes that apply to every similar title in the UI, everything that defines the title as a unit. I usually place there all attributes that are inside the border in the [CSS Box Model](https://www.w3schools.com/css/css_boxmodel.asp). Examples: `font-size`, `width`, `color`, `background`...

2. `top-bar__title` css class is used to apply styles that apply to this atom when its part of the `top-bar` molecule, when it is in that context. I usually place there attributes such as: `margin`, `position`, `top`, `right`, `tranlate` and attributes that I wan't to overwrite like: `display`, `width`...

Taking this into account our style would become:

```css
.top-bar__title {
  align-self: center;
  justify-self: auto;
  margin: 0 auto;
}

.title-3 {
  font-size: 18px;
  font-weight: 600;
}
```

Also we have to take into consideration that `title-3` is an atom and `top-bar__title` is a class for a molecule. Since we use molecule classes to overwrite atoms styles in a certain context we should place them above to take advantage of the 'cascading'. Pleaase checkout [this docs explaining cascading](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance).

```css
/** Atoms **/
.title-3 {
  font-size: 18px;
  font-weight: 600;
}

/** Molecules **/
.top-bar__title {
  align-self: center;
  justify-self: auto;
  margin: 0 auto;
}
```